### PR TITLE
Rewrite CSES Flight Routes Check Proof

### PR DESCRIPTION
**Pull request by izhang05**
I think writing the proof without the unnecessary contradiction makes it a lot clearer and easier to read.

_If any of the below doesn't apply to this Pull Request, mark the checkbox as done._

- [ ] I have tested my code.
- [ ] I have added my solution according to the steps [here](https://usaco.guide/general/adding-solution#steps).
- [ ] I have followed the code conventions mentioned [here](https://usaco.guide/general/adding-solution/#code-conventions), which include the following: 
  - I understand that if it is clear that I have not attempted to follow these guidelines (ex. if I have not used tabs to indent), my PR will be closed.
  - If changes are requested, I will re-request the review after making them.

### CODE REVIEW

File: solutions/cses-1682.mdx [hunk 1]
```diff
@@ -27,14 +27,14 @@ $1$ $x$ if you're running the DFS on $G$ and $x$ $1$ otherwise.
...
 
-Let's do a proof by contradiction. Assume that $\texttt{can[1][x]}$ is true for
-all vertices $x$ in both $G$ and $G'$, and there exists a pair of vertices
-$u, v$ such that $\texttt{can[u][v]} = \texttt{false}$. Since
-$\texttt{can[1][u]}$ is true in $G'$, then $\texttt{can[u][1]}$ must be true in
-$G$. Additionally, $\texttt{can[1][v]}$ must be true in $G$. So, you can travel
-from $u \rightarrow 1 \rightarrow v$, which contradicts the statement that
-$\texttt{can[u][v]} = \texttt{false}$. Thus, $\texttt{can[u][v]}$ is true for
-all vertices $u, v$.
+We will show that if $\texttt{can[1][x]}$ for $1 \leq x \leq n$ in both $G$ and $G'$,
```
Comment (SansPapyrus683) [inline solutions/cses-1682.mdx:30]: don't think it's good to abuse latex
can you explain this in layman terms?

Comment (izhang05) [reply solutions/cses-1682.mdx:30]: Yeah I agree, I just used that because it's the notation used in the first half of the solution.
Should I also rewrite the first part of the solution then?

**Comment by chezbgone** [comment]
From Discord discussion: Is the proof even necessary? I feel like adding an extra sentence to the first line of the "Main Idea" section would be better. The sentence can be something like "This is because if we want to find a path `u -> v` (u \to v), we can compose the two paths `u -> x` and `x -> v`.

**Comment by izhang05** [comment]
Yeah, in that case I'm not sure we need the can[u][v] either. It would probably be clearer to say something like "If for every node $x$, there is a path from $1$ to $x$ and a path from $x$ to $1$..."
